### DISCUSSION TITLE
Why AppImage for Linux?

### DISCUSSION THREAD

**Discussion by DeltaEpsilon19498** [TryQuiet/quiet#2235] [General]
Why is an appimage your only offering for linux? Why do you prefer it over flatpak, which allows sandboxing?

Also, how do I get packages for apt, rpm, aur, portage, etc.?

Suppose I want to compile the program myself, in order to not have to trust your binary releases perfectly correspond to the source code here (without any secret backdoors). Where are the build instructions for doing this?

Thank you for your help.

**Comment by Jacobthegr8** [comment]
I'm just glad an appimage is available. It makes it easier for a novice linux user to use the program, like me. Flatpak would be nice too for auto updates but that could be a good future addition.
